typings/Calender.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "globalize";
+declare module "react-dnd";
+declare module "react-dnd-html5-backend";
+declare module "react-big-calendar/lib/addons/dragAndDrop";
+declare module "react-big-calendar/lib/localizers/globalize";
```

Comment:
You'll eventually have to add actual typing for all these... but perhaps wait off till Andries gets the go-ahead